charts/coredns/coredns-pod-delete/engine.yaml

```diff
@@ -14,8 +14,7 @@ spec:
   auxiliaryAppInfo: ''
   components:
     runner:
-       image: 'litmuschaos/chaos-executor:1.0.0'
-       type: 'go'
+       image: 'litmuschaos/chaos-executor:latest'
```

Comment:
Can you please add type field  here ( mention a comment stating that it can be ansible/go)

Reply:
Pls, add engineState field also (it can support active and stop values). Do the same for every file